lib/util/lib/perfc.c

```diff
@@ -684,10 +687,10 @@ perfc_alloc_impl(
             err = merr(rc);
     }
 
-  errout:
+errout:
     if (err) {
-        log_warnx("unable to alloc perf counter %s/%s/%s from %s:%d",
-                  err, group, family, ctrseti_name, file, line);
+        log_warnx("unable to alloc perf counter %s/%s/%s from %s:%d", err, group, family,
+                ctrseti_name, file, line);
```

Comment:
We can adopt a convention to prevent mucking w/ newines (lnl = leave new line):
```
        log_warnx(
            "unable to alloc perf counter %s/%s/%s from %s:%d", // lnl
            err, group, family, ctrseti_name, file, line);
```

Reply:
yeah, i like to use "// dnl", from m4 :-)